code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -26,12 +26,14 @@ var/list/special_fruits = list()
 		if(initial(G.hydroflags) & filter)
 			. += T
 
-/obj/item/weapon/reagent_containers/food/snacks/grown/New(atom/loc, custom_plantname, mob/harvester)
+/obj/item/weapon/reagent_containers/food/snacks/grown/New(atom/loc, custom_plantname, mob/harvester, add_process=0)
 	..()
 	if(custom_plantname)
 		plantname = custom_plantname
 	if(ticker)
 		initialize(harvester)
+	if(add_process)
+		processing_objects.Add(src)
```

Comment:
given that botany is best known for producing loads of plants, particularly with some tweaks, this has potential to be performance impacting since you can make fucktons of processing objects very quickly; it's not something I would bar a merge over but we need to keep an eye on it

Reply:
Is there another way to do this besides using the processing list or is it just an issue with the nature of the feature?

Reply:
It's just in its nature, unless you want to pulse less often, but I don't think you do. This is just an inherent thing to consider when you make a call every 2 seconds on something that can be made in bulk

Reply:
A high 'tism splice can shit 13-15 plants every 20 seconds or so (even faster if you take the time to mutate, but no-one does). Plant 10 of this splice, get 390-450 crops each minute. You can reach, and far surpass those numbers when doing high intensity botany autismo, such as extreme crop farming with selfharvest (or minerals farming), or high yield luck farming via clovers. Keep in mind that that would be the tismo splice, not the grass + chanterelle splice, which is faster, but way less efficient, and the more used splice setup.
Test how laggy it'd get with 400 process crops. Then 800, etc. Then we can see what would be a sane limit for the process checks.

Reply:
I suggest, for testing, that you VV a plasma cabbage seed, go down to the seed var, access it, set it to potency 200, yield 50 and production 1, then plant. That way you can easily spawn the crops in 50 crop chunks in the "natural" (not adminspawned) way they'd spawn in game.